features/scroll-markers.yml

```
@@ -0,0 +1,4 @@
+name: Scroll markers
+description: The `::scroll-marker` CSS pseudo-element is a focusable control within its ancestor scroll container's `::scroll-marker-group` that scrolls to its originating element.
```

I'm having trouble understanding this. Would you mind giving me a practical example of how this might work/what this would look like? This would help me understand, and potentially propose a slight rewrite of this.

---

If you have chrome canary with experimental web platform features enabled, there are some great demos at:
* https://chrome.dev/carousel-configurator/ (turn on "Dot Navigation" switch) which shows the CSS used to produce the scroll markers, and
* https://chrome.dev/carousel/ which has many demos using scroll markers to create dots to navigate to particular targets.

As an example, the following creates a set of circular "markers" to scroll to each of the slides in a slideshow:
```html
<style>
  .slides {
    overflow: auto;
    scroll-snap-type: x mandatory;
    white-space: nowrap;
    scroll-marker-group: after;
    counter-reset: slide-number;
  }
  .slide::scroll-marker {
    counter-increment: slide-number;
    content: counter(slide-number);
    border-radius: 50%;
    border: 1px solid black;
  }
}
</style>
<div class=slides>
  <div class=slide></div>
  <div class=slide></div>
  <div class=slide></div>
  <div class=slide></div>
  <div class=slide></div>
<div class=slideshow>
```

Here's a somewhat simple codepen trying to show the same: https://codepen.io/flackr/pen/MWMLPpP

---

Now that I've read the spec a bit more, I'd like to propose different alternatives:

```suggestion
description: The `::scroll-marker-group` CSS pseudo-element contains scroll markers (stylable with the `::scroll-marker` pseudo-element), and appears either immediately before or after a scroll container.
```

Or 

```suggestion
description: A scroll container can be navigated by activating `::scroll-marker` pseudo-elements which appear in a generated `::scroll-marker-group` pseudo-element, either before or after the scroll container.
```